docs/section11/Rule_11-1.md

```diff
@@ -0,0 +1,64 @@
+# Service_Water_Heating - Rule 11-1
+**Schema Version:** 0.0.36  
+
+**Mandatory Rule:** TRUE
+
+**Rule ID:** 11-1
+
+**Rule Description:** "Where a complete service water-heating system exists, the proposed design shall reflect the actual system type. Where a service water-heating system has been designed the service waterheating type shall be consistent with design documents."
+
+**Rule Assertion:** Options are PASS/FAIL
+
+**Appendix G Section Reference:** Table G3.1 #11, proposed column, a & b
+
+**Evaluation Context:** P-RMD each SWH Distribution
+**Data Lookup:**   
+**Function Call:** 
+- **get_component_by_id**
+- **compare_context_pair** - there is no RDS for this function, but it is a function developed for Rule 1-6 that compares two elements
+- **get_SWH_equipment_associated_with_each_swh_distribution_system**  
+
+**Applicability Checks:**
+- check that the U_RMD has SWH Uses with loads
+
+## Applicability Checks:
+- only projects with SWH in the user model are expected to have a SWH system in the proposed model.  If there is no SWH in the user model, we assume there is no SWH system designed, and instead rule 11-3 Applies and P_RMD matches B_RMD.
+- use the function get_SWH_equipment_associated_with_each_swh_distribution_system to get a dictionary of SWH equipment associated with each Distribution System for the Proposed model (pumps, tanks, SWH use, distribution system, etc): `p_swh_system_and_equip_dict = get_SWH_equipment_associated_with_each_swh_distribution_system(P_RMD)`
+- use the function get_SWH_equipment_associated_with_each_swh_distribution_system to get a dictionary of SWH equipment associated with each Distribution System for the User model (pumps, tanks, SWH use, distribution system, etc): `u_swh_system_and_equip_dict = get_SWH_equipment_associated_with_each_swh_distribution_system(U_RMD)`
+- for each distribution system: `for distribution_id in p_swh_system_and_equip_dict:`
```

Comment:
If we are verifying that all equipment in the user model has matching equipment in the proposed model, should we be iterating through the user model instead of the proposed model?

Reply:
@JacksonJ-KC - I think we discussed this on a call, but I never responded formally.  There are advantages and disadvantages for both modes.  The way this rule and 11-3 are written, they both look at the **DistributionSytems** in the proposed model to confirm that each one is modeled correctly.

However, looking over this again, this misses the case where a distribution system is included in the user model but not the proposed model, so I've added a check for (and give a FAIL result) for this case